gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -220,6 +220,8 @@ email:
   from: noreply@my.domain
   username: user@my.domain
   password: password
+  allowlist:
+    - "*@*.*"
```

Comment:
see comments on the [gravitee.yaml](https://github.com/gravitee-io/gravitee-access-management/pull/2849/files#diff-12f73c7e177a2eda64da20512fd027158e798e496fabb691acf7aa8adb302b40) in the management-api service